media-tv/plex-media-server/plex-media-server-1.22.2.4282.ebuild

```
@@ -0,0 +1,128 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit pax-utils systemd unpacker
```

We don't seem to be actually using anything from [pax-utils](https://devmanual.gentoo.org/eclass-reference/pax-utils.eclass/index.html) right now?

```suggestion
inherit systemd unpacker
```